unity/Holo/Assets/Editor/ModelPreprocessing/AssetBundleHandling/AssetBundleCreator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+class AssetBundleCreator
+{
+    public GameObject ModelGameObject { get; set; }
+    public Mesh Mesh { get; set; }
+
+    private string rootAssetsDir;
+    private List<string> assetsPath = new List<string>();

```

Comment:
Maybe it would be cleaner and easier to use `Dictionary` class for storing these paths?
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2?redirectedfrom=MSDN&view=netframework-4.7.2